terraform/prod-lon.vpc_peering.json

```diff
@@ -23,6 +23,12 @@
         "vpc_id": "vpc-074a7e7c7b4eccc8c",
         "subnet_cidr": "172.18.0.0/22"
     },
+    {
+        "peer_name": "dit-staging_datasci-staging",
```

Comment:
The subnet CIDR looks right to me, but could you swap it with the next definition?  It'd be good to keep the list ordered numerically by subnet CIDR.

Reply:
Done